metadata-ingestion/src/datahub/ingestion/source/thrift/thrift.py

```diff
@@ -743,8 +840,8 @@ def bind_Type_from_qualified_name(self, qualified_name: str) -> Type:
 
         func = {
             ResolvedEnum: lambda _: int,
-            ResolvedUnion: lambda _: None,  # TODO 29: Container container = { "mesos": {} }
-            ResolvedStruct: lambda _: None,  # TODO 4: optional AdditionalParams additionalParams = {};
+            ResolvedUnion: lambda _: Union,
+            ResolvedStruct: lambda _: Tuple,
```

Comment:
Let's not use `Union` or `Tuple`
Since we don't want to do type checking, means we use this Type to validate the default value, we can create a new type.
```
from typing import NewType

SkipValidationType = NewType('SkipValidationType', str)
```
and 
```
ResolvedUnion: lambda _: SkipValidationType
ResolvedStruct: lambda _: SkipValidationType
```